jwst/regtest/test_nircam_tsimg.py

```diff
@@ -60,3 +60,28 @@ def test_nircam_tsimage_stage3_phot(run_pipelines):
 
     # setdiff returns a table of length zero if there is no difference
     assert len(setdiff(table, table_truth)) == 0
+
+
+@pytest.mark.bigdata
+def test_nircam_setpointing(_jail, rtdata, fitsdiff_default_kwargs):
+    """
+    Regression test of the set_telescope_pointing script on a level-1b
+    NIRCam TSO imaging file.
+    """
+    # Get SIAF PRD database file
+    siaf_path = rtdata.get_data("common/prd.db")
+    rtdata.get_data("nircam/tsimg/jw00312006001_02102_00001-seg001_nrcb1_uncal.fits")
+    # The add_wcs function overwrites its input, so output = input
+    rtdata.output = rtdata.input
+
+    # Call the WCS routine, using the ENGDB_Service
+    try:
+        add_wcs(rtdata.input, siaf_path=siaf_path)
+    except ValueError:
+        pytest.skip('Engineering Database not available.')
```

Comment:
Is the Engineering database sometimes not available?  Do we know how often? Is mocking it overkill?

Reply:
I don't recall seeing any instances of it *not* being available. @stscieisenhamer might have better info.